docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -51,6 +51,8 @@ Use these controls to filter what's included in the logs table:
 
 * The *Show metrics columns* toggle includes more or less data in the table, pertaining to the timing of each rule execution.
 
+* The *Actions* column allows you to show the alerts generated from a given rule execution. Click the filter icon (image:images/filter-icon.png[Filter icon,18,17]) to create a global search filter based on the rule execution's ID value. This replaces any previously applied filters, and changes the global date and time picker to 24 hours before and after the rule execution.
```

Comment:
```suggestion
* The *Actions* column allows you to show the alerts generated from a given rule execution. Click the filter icon (image:images/filter-icon.png[Filter icon,18,17]) to create a global search filter based on the rule's execution ID. This replaces any previously applied filters, and changes the global date and time range to 24 hours before and after the rule execution.
```

Reply:
The last sentence makes sense, but I had to test it to visualize what the new time range would be and to see whether it'd affect the rule execution log's time range (it didn't). Normally I'd suggest adding an example, but I don't think the scenario you described is complex enough to warrant it (not like our look back time mechanism). One thing I am wondering: do you think it'd be worth mentioning that users can revert to the global filters they were formerly using by clicking **Restore previous filters** in the confirmation toast?
<img width="449" alt="Screen Shot 2022-06-06 at 5 23 09 PM" src="https://user-images.githubusercontent.com/79124755/172251660-7e7c639a-1013-409e-941a-c585b9a04c78.png">

Reply:
Ah, gotcha @joepeeples. Makes sense to keep rule execution's ID as is then. I'll edit my comment remove that suggestion.